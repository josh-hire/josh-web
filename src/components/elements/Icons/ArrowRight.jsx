import React from 'react';

export const IconArrowRight = () => {
  return (
    <svg fill="none" height="12" viewBox="0 0 12 12" width="12" xmlns="http://www.w3.org/2000/svg">
      <path d="M4.5 9L7.5 6L4.5 3" stroke="#A0A0AB" strokeLinecap="round" strokeLinejoin="round"/>
    </svg>
  );
};
