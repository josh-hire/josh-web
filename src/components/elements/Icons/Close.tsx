import React from 'react';

export const IconClose = () => {
  return (
    <svg fill="none" height="21" viewBox="0 0 20 21" width="20" xmlns="http://www.w3.org/2000/svg">
      <path d="M14.999 5.5L4.99902 15.5" stroke="#26272B" strokeLinecap="round" strokeLinejoin="round" strokeWidth="2.08"/>
      <path d="M4.99902 5.5L14.999 15.5" stroke="#26272B" strokeLinecap="round" strokeLinejoin="round" strokeWidth="2.08"/>
    </svg>

  );
};
