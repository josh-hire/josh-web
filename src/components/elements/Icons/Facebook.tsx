import React from 'react';

export const IconFacebook = () => {
  return (
    <svg fill="none" height="46" viewBox="0 0 46 46" width="46" xmlns="http://www.w3.org/2000/svg">
      <path clipRule="evenodd" d="M1 23C1 10.8497 10.8497 1 23 1C35.1503 1 45 10.8497 45 23C45 35.1503 35.1503 45 23 45C10.8497 45 1 35.1503 1 23Z" fill="white" fillRule="evenodd" stroke="#C7C7C7" strokeWidth="0.5"/>
      <path d="M25.5625 16.125H29V12H25.5625C22.9088 12 20.75 14.1588 20.75 16.8125V18.875H18V23H20.75V34H24.875V23H28.3125L29 18.875H24.875V16.8125C24.875 16.4399 25.1899 16.125 25.5625 16.125Z" fill="#70707B"/>
    </svg>
  );
};
