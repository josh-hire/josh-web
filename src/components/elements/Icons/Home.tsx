import React from 'react';

export const IconHome = () => {
  return (
    <svg fill="none" height="14" viewBox="0 0 14 14" width="14" xmlns="http://www.w3.org/2000/svg">
      <path d="M9.84405 12.3348H9.84393C9.7992 12.3348 9.75473 12.3259 9.71302 12.3083L9.29482 13.3003L9.71302 12.3083C9.6713 12.2907 9.63294 12.2647 9.6004 12.2315C9.56783 12.1982 9.54175 12.1582 9.52406 12.1138C9.50636 12.0694 9.49752 12.0216 9.49824 11.9733L9.49836 11.9652V11.9572V11.1693C9.49836 10.5063 9.237 9.86894 8.76927 9.39779C8.30128 8.92636 7.66487 8.66008 6.99961 8.66008C6.33435 8.66008 5.69794 8.92636 5.22995 9.39779C4.76222 9.86894 4.50086 10.5063 4.50086 11.1693V11.9572C4.50086 12.0542 4.46254 12.1456 4.39683 12.2118C4.33138 12.2777 4.2443 12.3133 4.15516 12.3133H2.02183C1.9327 12.3133 1.84561 12.2777 1.78017 12.2118C1.71446 12.1456 1.67614 12.0542 1.67614 11.9572V5.52355C1.67805 5.47159 1.69098 5.4209 1.71379 5.37495C1.73753 5.32712 1.77122 5.28591 1.81188 5.25394C1.81193 5.2539 1.81199 5.25385 1.81205 5.25381L6.7893 1.35025L6.79002 1.34968C6.85083 1.30191 6.92461 1.27673 6.99961 1.27673C7.0746 1.27673 7.14839 1.30191 7.2092 1.34968L7.20992 1.35025L12.187 5.25367C12.187 5.25369 12.187 5.25371 12.1871 5.25373C12.2285 5.28626 12.2626 5.32835 12.2864 5.37723C12.3101 5.42579 12.3227 5.4795 12.3231 5.53427V11.9787C12.3231 12.0757 12.2848 12.1671 12.2191 12.2333C12.1536 12.2992 12.0665 12.3348 11.9774 12.3348H9.84405Z" fill="#E8590C" stroke="#E8590C" strokeWidth="2.15306"/>
    </svg>
  );
};
