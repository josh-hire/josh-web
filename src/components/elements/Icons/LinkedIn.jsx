import React from 'react';

export const IconLinkedIn = () => {
  return (
    <svg fill="none" height="46" viewBox="0 0 46 46" width="46" xmlns="http://www.w3.org/2000/svg">
      <path clipRule="evenodd" d="M1 23C1 10.8497 10.8497 1 23 1C35.1503 1 45 10.8497 45 23C45 35.1503 35.1503 45 23 45C10.8497 45 1 35.1503 1 23Z" fill="white" fillRule="evenodd" stroke="#C7C7C7" strokeWidth="0.5"/>
      <path clipRule="evenodd" d="M20.429 19.969H24.143V21.819C24.678 20.755 26.05 19.799 28.111 19.799C32.062 19.799 33 21.917 33 25.803V33H29V26.688C29 24.475 28.465 23.227 27.103 23.227C25.214 23.227 24.429 24.572 24.429 26.687V33H20.429V19.969ZM13.57 32.83H17.57V19.799H13.57V32.83ZM18.143 15.55C18.1431 15.8853 18.0767 16.2172 17.9474 16.5266C17.8181 16.8359 17.6287 17.1165 17.39 17.352C16.9064 17.8326 16.2518 18.1017 15.57 18.1C14.8894 18.0995 14.2363 17.8312 13.752 17.353C13.5142 17.1167 13.3254 16.8358 13.1963 16.5264C13.0673 16.217 13.0006 15.8852 13 15.55C13 14.873 13.27 14.225 13.753 13.747C14.2369 13.2682 14.8902 12.9997 15.571 13C16.253 13 16.907 13.269 17.39 13.747C17.872 14.225 18.143 14.873 18.143 15.55Z" fill="#70707B" fillRule="evenodd"/>
    </svg>
  );
};
