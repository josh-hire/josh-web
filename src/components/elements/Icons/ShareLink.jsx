import React from 'react';

export const IconShareLink = () => {
  return (
    <svg fill="none" height="46" viewBox="0 0 46 46" width="46" xmlns="http://www.w3.org/2000/svg">
      <path clipRule="evenodd" d="M1 23C1 10.8497 10.8497 1 23 1C35.1503 1 45 10.8497 45 23C45 35.1503 35.1503 45 23 45C10.8497 45 1 35.1503 1 23Z" fill="white" fillRule="evenodd" stroke="#C7C7C7" strokeWidth="0.5"/>
      <path d="M20.1753 29.1919L19.1061 30.2545C18.6582 30.6954 18.0549 30.9425 17.4264 30.9425C16.7979 30.9425 16.1946 30.6954 15.7467 30.2545C15.5283 30.0383 15.355 29.7809 15.2366 29.4972C15.1183 29.2135 15.0574 28.9092 15.0574 28.6019C15.0574 28.2945 15.1183 27.9902 15.2366 27.7066C15.355 27.4229 15.5283 27.1655 15.7467 26.9492L19.6803 23.0486C20.4947 22.2395 22.0286 21.0488 23.1466 22.1576C23.2669 22.2859 23.4119 22.3887 23.5728 22.4597C23.7337 22.5308 23.9072 22.5687 24.0831 22.5712C24.259 22.5737 24.4335 22.5408 24.5964 22.4743C24.7592 22.4079 24.907 22.3093 25.031 22.1845C25.1549 22.0597 25.2524 21.9112 25.3177 21.7479C25.383 21.5845 25.4147 21.4097 25.4109 21.2339C25.4071 21.058 25.368 20.8848 25.2958 20.7244C25.2236 20.564 25.1198 20.4198 24.9907 20.3004C23.0925 18.4154 20.2835 18.7639 17.8376 21.1914L13.904 25.0933C13.4392 25.5515 13.0705 26.0979 12.8193 26.7003C12.5682 27.3027 12.4397 27.9492 12.4414 28.6019C12.4398 29.2545 12.5683 29.901 12.8194 30.5034C13.0706 31.1058 13.4393 31.6522 13.904 32.1104C14.8408 33.0397 16.1076 33.5599 17.4271 33.5572C18.7035 33.5572 19.9799 33.0754 20.9501 32.1104L22.0207 31.0478C22.1427 30.9271 22.2398 30.7835 22.3062 30.6251C22.3726 30.4668 22.4072 30.2969 22.4078 30.1252C22.4084 29.9535 22.3751 29.7834 22.3098 29.6246C22.2445 29.4658 22.1485 29.3214 22.0273 29.1998C21.7822 28.9541 21.4499 28.8154 21.1029 28.8139C20.7559 28.8124 20.4224 28.9483 20.1753 29.1919ZM32.0975 14.033C30.0555 12.0082 27.2017 11.8986 25.3141 13.773L23.9822 15.0956C23.7361 15.34 23.5971 15.6721 23.5959 16.0189C23.5946 16.3657 23.7312 16.6988 23.9756 16.945C24.2199 17.1911 24.5521 17.33 24.8989 17.3313C25.2457 17.3325 25.5788 17.1959 25.8249 16.9516L27.1581 15.6302C28.1349 14.6587 29.4166 15.0613 30.2548 15.8903C30.6983 16.3325 30.9439 16.9186 30.9439 17.5429C30.9439 18.1673 30.6983 18.7534 30.2548 19.1942L26.0572 23.3575C24.1379 25.2596 23.2377 24.3686 22.8536 23.9872C22.7317 23.8662 22.5872 23.7704 22.4283 23.7052C22.2694 23.6401 22.0993 23.6068 21.9276 23.6074C21.7558 23.6081 21.5859 23.6425 21.4275 23.7088C21.2691 23.7751 21.1253 23.8719 21.0043 23.9938C20.8833 24.1156 20.7875 24.2601 20.7223 24.419C20.6572 24.5779 20.6239 24.7481 20.6246 24.9198C20.6252 25.0915 20.6596 25.2614 20.7259 25.4199C20.7922 25.5783 20.889 25.7221 21.0109 25.8431C21.8926 26.7169 22.8985 27.1499 23.9518 27.1499C25.2415 27.1499 26.605 26.5004 27.9026 25.2134L32.1002 21.0515C32.5639 20.5926 32.9319 20.0461 33.1828 19.4438C33.4336 18.8415 33.5623 18.1954 33.5614 17.5429C33.5626 16.89 33.4338 16.2434 33.1824 15.6408C32.9311 15.0382 32.5623 14.4916 32.0975 14.033Z" fill="#70707B"/>
    </svg>
  );
};
