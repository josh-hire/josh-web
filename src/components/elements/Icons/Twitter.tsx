import React from 'react';

export const IconTwitter = () => {
  return (
    <svg fill="none" height="46" viewBox="0 0 46 46" width="46" xmlns="http://www.w3.org/2000/svg">
      <path clipRule="evenodd" d="M1 23C1 10.8497 10.8497 1 23 1C35.1503 1 45 10.8497 45 23C45 35.1503 35.1503 45 23 45C10.8497 45 1 35.1503 1 23Z" fill="white" fillRule="evenodd" stroke="#C7C7C7" strokeWidth="0.5"/>
      <path clipRule="evenodd" d="M23.1967 19.2563L23.2422 19.9483L22.4835 19.8636C19.7219 19.5387 17.3093 18.4371 15.2609 16.5869L14.2594 15.6689L14.0015 16.3468C13.4552 17.858 13.8042 19.454 14.9422 20.5274C15.5492 21.1205 15.4126 21.2053 14.3656 20.8522C14.0015 20.7392 13.6828 20.6545 13.6525 20.6968C13.5463 20.7957 13.9104 22.0809 14.1987 22.5894C14.5932 23.2955 15.3974 23.9876 16.2775 24.3972L17.021 24.722L16.1409 24.7361C15.2912 24.7361 15.2609 24.7502 15.3519 25.0468C15.6554 25.9649 16.8541 26.9394 18.1894 27.3631L19.1302 27.6597L18.3108 28.1116C17.0969 28.7613 15.6706 29.1285 14.2442 29.1567C13.5614 29.1709 13 29.2273 13 29.2697C13 29.411 14.8512 30.2019 15.9285 30.5126C19.1605 31.4306 22.9994 31.0351 25.8824 29.4674C27.9309 28.3517 29.9793 26.1343 30.9353 23.9876C31.4512 22.8436 31.9671 20.7533 31.9671 19.7506C31.9671 19.1009 32.0126 19.0162 32.8623 18.2394C33.363 17.7874 33.8334 17.2931 33.9245 17.1519C34.0762 16.8835 34.061 16.8835 33.2872 17.1236C31.9974 17.5473 31.8153 17.4908 32.4526 16.8553C32.923 16.4033 33.4844 15.5842 33.4844 15.3441C33.4844 15.3017 33.2568 15.3723 32.9989 15.4994C32.7258 15.6407 32.1188 15.8525 31.6636 15.9796L30.8442 16.2197L30.1007 15.7537C29.691 15.4994 29.1144 15.217 28.811 15.1322C28.0371 14.9345 26.8535 14.9627 26.1556 15.1887C24.2589 15.8243 23.0601 17.4626 23.1967 19.2563Z" fill="#70707B" fillRule="evenodd"/>
    </svg>
  );
};
