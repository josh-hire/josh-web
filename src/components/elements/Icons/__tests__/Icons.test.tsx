import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { IconArrowRight } from '../ArrowRight';
import { IconClose } from '../Close';
import { IconFacebook } from '../Facebook';
import { IconHome } from '../Home';
import { IconLinkedIn } from '../LinkedIn';
import { IconShareLink } from '../ShareLink';
import { IconTwitter } from '../Twitter';

describe('src/components/elements/Icons', ()=>{
  test('IconArrowRight',()=>{
    const { asFragment } = render(<IconArrowRight />);
    expect(asFragment()).toMatchSnapshot();
  });
  test('IconClose',()=>{
    const { asFragment } = render(<IconClose />);
    expect(asFragment()).toMatchSnapshot();
  });

  test('IconFacebook',()=>{
    const { asFragment } = render(<IconFacebook />);
    expect(asFragment()).toMatchSnapshot();
  });

  test('IconHome',()=>{
    const { asFragment } = render(<IconHome />);
    expect(asFragment()).toMatchSnapshot();
  });

  test('IconLinkedIn',()=>{
    const { asFragment } = render(<IconLinkedIn />);
    expect(asFragment()).toMatchSnapshot();
  });

  test('IconShareLink',()=>{
    const { asFragment } = render(<IconShareLink />);
    expect(asFragment()).toMatchSnapshot();
  });

  test('IconTwitter',()=>{
    const { asFragment } = render(<IconTwitter />);
    expect(asFragment()).toMatchSnapshot();
  });
});
