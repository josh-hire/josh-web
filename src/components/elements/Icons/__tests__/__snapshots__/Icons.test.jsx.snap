// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`src/components/elements/Icons IconArrowRight 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    height="12"
    viewBox="0 0 12 12"
    width="12"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M4.5 9L7.5 6L4.5 3"
      stroke="#A0A0AB"
      stroke-linecap="round"
      stroke-linejoin="round"
    />
  </svg>
</DocumentFragment>
`;

exports[`src/components/elements/Icons IconClose 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    height="21"
    viewBox="0 0 20 21"
    width="20"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M14.999 5.5L4.99902 15.5"
      stroke="#26272B"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2.08"
    />
    <path
      d="M4.99902 5.5L14.999 15.5"
      stroke="#26272B"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2.08"
    />
  </svg>
</DocumentFragment>
`;

exports[`src/components/elements/Icons IconFacebook 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    height="46"
    viewBox="0 0 46 46"
    width="46"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M1 23C1 10.8497 10.8497 1 23 1C35.1503 1 45 10.8497 45 23C45 35.1503 35.1503 45 23 45C10.8497 45 1 35.1503 1 23Z"
      fill="white"
      fill-rule="evenodd"
      stroke="#C7C7C7"
      stroke-width="0.5"
    />
    <path
      d="M25.5625 16.125H29V12H25.5625C22.9088 12 20.75 14.1588 20.75 16.8125V18.875H18V23H20.75V34H24.875V23H28.3125L29 18.875H24.875V16.8125C24.875 16.4399 25.1899 16.125 25.5625 16.125Z"
      fill="#70707B"
    />
  </svg>
</DocumentFragment>
`;

exports[`src/components/elements/Icons IconHome 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    height="14"
    viewBox="0 0 14 14"
    width="14"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M9.84405 12.3348H9.84393C9.7992 12.3348 9.75473 12.3259 9.71302 12.3083L9.29482 13.3003L9.71302 12.3083C9.6713 12.2907 9.63294 12.2647 9.6004 12.2315C9.56783 12.1982 9.54175 12.1582 9.52406 12.1138C9.50636 12.0694 9.49752 12.0216 9.49824 11.9733L9.49836 11.9652V11.9572V11.1693C9.49836 10.5063 9.237 9.86894 8.76927 9.39779C8.30128 8.92636 7.66487 8.66008 6.99961 8.66008C6.33435 8.66008 5.69794 8.92636 5.22995 9.39779C4.76222 9.86894 4.50086 10.5063 4.50086 11.1693V11.9572C4.50086 12.0542 4.46254 12.1456 4.39683 12.2118C4.33138 12.2777 4.2443 12.3133 4.15516 12.3133H2.02183C1.9327 12.3133 1.84561 12.2777 1.78017 12.2118C1.71446 12.1456 1.67614 12.0542 1.67614 11.9572V5.52355C1.67805 5.47159 1.69098 5.4209 1.71379 5.37495C1.73753 5.32712 1.77122 5.28591 1.81188 5.25394C1.81193 5.2539 1.81199 5.25385 1.81205 5.25381L6.7893 1.35025L6.79002 1.34968C6.85083 1.30191 6.92461 1.27673 6.99961 1.27673C7.0746 1.27673 7.14839 1.30191 7.2092 1.34968L7.20992 1.35025L12.187 5.25367C12.187 5.25369 12.187 5.25371 12.1871 5.25373C12.2285 5.28626 12.2626 5.32835 12.2864 5.37723C12.3101 5.42579 12.3227 5.4795 12.3231 5.53427V11.9787C12.3231 12.0757 12.2848 12.1671 12.2191 12.2333C12.1536 12.2992 12.0665 12.3348 11.9774 12.3348H9.84405Z"
      fill="#E8590C"
      stroke="#E8590C"
      stroke-width="2.15306"
    />
  </svg>
</DocumentFragment>
`;

exports[`src/components/elements/Icons IconLinkedIn 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    height="46"
    viewBox="0 0 46 46"
    width="46"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M1 23C1 10.8497 10.8497 1 23 1C35.1503 1 45 10.8497 45 23C45 35.1503 35.1503 45 23 45C10.8497 45 1 35.1503 1 23Z"
      fill="white"
      fill-rule="evenodd"
      stroke="#C7C7C7"
      stroke-width="0.5"
    />
    <path
      clip-rule="evenodd"
      d="M20.429 19.969H24.143V21.819C24.678 20.755 26.05 19.799 28.111 19.799C32.062 19.799 33 21.917 33 25.803V33H29V26.688C29 24.475 28.465 23.227 27.103 23.227C25.214 23.227 24.429 24.572 24.429 26.687V33H20.429V19.969ZM13.57 32.83H17.57V19.799H13.57V32.83ZM18.143 15.55C18.1431 15.8853 18.0767 16.2172 17.9474 16.5266C17.8181 16.8359 17.6287 17.1165 17.39 17.352C16.9064 17.8326 16.2518 18.1017 15.57 18.1C14.8894 18.0995 14.2363 17.8312 13.752 17.353C13.5142 17.1167 13.3254 16.8358 13.1963 16.5264C13.0673 16.217 13.0006 15.8852 13 15.55C13 14.873 13.27 14.225 13.753 13.747C14.2369 13.2682 14.8902 12.9997 15.571 13C16.253 13 16.907 13.269 17.39 13.747C17.872 14.225 18.143 14.873 18.143 15.55Z"
      fill="#70707B"
      fill-rule="evenodd"
    />
  </svg>
</DocumentFragment>
`;

exports[`src/components/elements/Icons IconSearch 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    height="24"
    viewBox="0 0 24 24"
    width="24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M11 19C15.4183 19 19 15.4183 19 11C19 6.58172 15.4183 3 11 3C6.58172 3 3 6.58172 3 11C3 15.4183 6.58172 19 11 19Z"
      stroke="#2E3032"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2.5"
    />
    <path
      d="M21.0004 21.0004L16.6504 16.6504"
      stroke="#2E3032"
      stroke-linecap="round"
      stroke-linejoin="round"
      stroke-width="2.5"
    />
  </svg>
</DocumentFragment>
`;

exports[`src/components/elements/Icons IconShareLink 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    height="46"
    viewBox="0 0 46 46"
    width="46"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M1 23C1 10.8497 10.8497 1 23 1C35.1503 1 45 10.8497 45 23C45 35.1503 35.1503 45 23 45C10.8497 45 1 35.1503 1 23Z"
      fill="white"
      fill-rule="evenodd"
      stroke="#C7C7C7"
      stroke-width="0.5"
    />
    <path
      d="M20.1753 29.1919L19.1061 30.2545C18.6582 30.6954 18.0549 30.9425 17.4264 30.9425C16.7979 30.9425 16.1946 30.6954 15.7467 30.2545C15.5283 30.0383 15.355 29.7809 15.2366 29.4972C15.1183 29.2135 15.0574 28.9092 15.0574 28.6019C15.0574 28.2945 15.1183 27.9902 15.2366 27.7066C15.355 27.4229 15.5283 27.1655 15.7467 26.9492L19.6803 23.0486C20.4947 22.2395 22.0286 21.0488 23.1466 22.1576C23.2669 22.2859 23.4119 22.3887 23.5728 22.4597C23.7337 22.5308 23.9072 22.5687 24.0831 22.5712C24.259 22.5737 24.4335 22.5408 24.5964 22.4743C24.7592 22.4079 24.907 22.3093 25.031 22.1845C25.1549 22.0597 25.2524 21.9112 25.3177 21.7479C25.383 21.5845 25.4147 21.4097 25.4109 21.2339C25.4071 21.058 25.368 20.8848 25.2958 20.7244C25.2236 20.564 25.1198 20.4198 24.9907 20.3004C23.0925 18.4154 20.2835 18.7639 17.8376 21.1914L13.904 25.0933C13.4392 25.5515 13.0705 26.0979 12.8193 26.7003C12.5682 27.3027 12.4397 27.9492 12.4414 28.6019C12.4398 29.2545 12.5683 29.901 12.8194 30.5034C13.0706 31.1058 13.4393 31.6522 13.904 32.1104C14.8408 33.0397 16.1076 33.5599 17.4271 33.5572C18.7035 33.5572 19.9799 33.0754 20.9501 32.1104L22.0207 31.0478C22.1427 30.9271 22.2398 30.7835 22.3062 30.6251C22.3726 30.4668 22.4072 30.2969 22.4078 30.1252C22.4084 29.9535 22.3751 29.7834 22.3098 29.6246C22.2445 29.4658 22.1485 29.3214 22.0273 29.1998C21.7822 28.9541 21.4499 28.8154 21.1029 28.8139C20.7559 28.8124 20.4224 28.9483 20.1753 29.1919ZM32.0975 14.033C30.0555 12.0082 27.2017 11.8986 25.3141 13.773L23.9822 15.0956C23.7361 15.34 23.5971 15.6721 23.5959 16.0189C23.5946 16.3657 23.7312 16.6988 23.9756 16.945C24.2199 17.1911 24.5521 17.33 24.8989 17.3313C25.2457 17.3325 25.5788 17.1959 25.8249 16.9516L27.1581 15.6302C28.1349 14.6587 29.4166 15.0613 30.2548 15.8903C30.6983 16.3325 30.9439 16.9186 30.9439 17.5429C30.9439 18.1673 30.6983 18.7534 30.2548 19.1942L26.0572 23.3575C24.1379 25.2596 23.2377 24.3686 22.8536 23.9872C22.7317 23.8662 22.5872 23.7704 22.4283 23.7052C22.2694 23.6401 22.0993 23.6068 21.9276 23.6074C21.7558 23.6081 21.5859 23.6425 21.4275 23.7088C21.2691 23.7751 21.1253 23.8719 21.0043 23.9938C20.8833 24.1156 20.7875 24.2601 20.7223 24.419C20.6572 24.5779 20.6239 24.7481 20.6246 24.9198C20.6252 25.0915 20.6596 25.2614 20.7259 25.4199C20.7922 25.5783 20.889 25.7221 21.0109 25.8431C21.8926 26.7169 22.8985 27.1499 23.9518 27.1499C25.2415 27.1499 26.605 26.5004 27.9026 25.2134L32.1002 21.0515C32.5639 20.5926 32.9319 20.0461 33.1828 19.4438C33.4336 18.8415 33.5623 18.1954 33.5614 17.5429C33.5626 16.89 33.4338 16.2434 33.1824 15.6408C32.9311 15.0382 32.5623 14.4916 32.0975 14.033Z"
      fill="#70707B"
    />
  </svg>
</DocumentFragment>
`;

exports[`src/components/elements/Icons IconTwitter 1`] = `
<DocumentFragment>
  <svg
    fill="none"
    height="46"
    viewBox="0 0 46 46"
    width="46"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M1 23C1 10.8497 10.8497 1 23 1C35.1503 1 45 10.8497 45 23C45 35.1503 35.1503 45 23 45C10.8497 45 1 35.1503 1 23Z"
      fill="white"
      fill-rule="evenodd"
      stroke="#C7C7C7"
      stroke-width="0.5"
    />
    <path
      clip-rule="evenodd"
      d="M23.1967 19.2563L23.2422 19.9483L22.4835 19.8636C19.7219 19.5387 17.3093 18.4371 15.2609 16.5869L14.2594 15.6689L14.0015 16.3468C13.4552 17.858 13.8042 19.454 14.9422 20.5274C15.5492 21.1205 15.4126 21.2053 14.3656 20.8522C14.0015 20.7392 13.6828 20.6545 13.6525 20.6968C13.5463 20.7957 13.9104 22.0809 14.1987 22.5894C14.5932 23.2955 15.3974 23.9876 16.2775 24.3972L17.021 24.722L16.1409 24.7361C15.2912 24.7361 15.2609 24.7502 15.3519 25.0468C15.6554 25.9649 16.8541 26.9394 18.1894 27.3631L19.1302 27.6597L18.3108 28.1116C17.0969 28.7613 15.6706 29.1285 14.2442 29.1567C13.5614 29.1709 13 29.2273 13 29.2697C13 29.411 14.8512 30.2019 15.9285 30.5126C19.1605 31.4306 22.9994 31.0351 25.8824 29.4674C27.9309 28.3517 29.9793 26.1343 30.9353 23.9876C31.4512 22.8436 31.9671 20.7533 31.9671 19.7506C31.9671 19.1009 32.0126 19.0162 32.8623 18.2394C33.363 17.7874 33.8334 17.2931 33.9245 17.1519C34.0762 16.8835 34.061 16.8835 33.2872 17.1236C31.9974 17.5473 31.8153 17.4908 32.4526 16.8553C32.923 16.4033 33.4844 15.5842 33.4844 15.3441C33.4844 15.3017 33.2568 15.3723 32.9989 15.4994C32.7258 15.6407 32.1188 15.8525 31.6636 15.9796L30.8442 16.2197L30.1007 15.7537C29.691 15.4994 29.1144 15.217 28.811 15.1322C28.0371 14.9345 26.8535 14.9627 26.1556 15.1887C24.2589 15.8243 23.0601 17.4626 23.1967 19.2563Z"
      fill="#70707B"
      fill-rule="evenodd"
    />
  </svg>
</DocumentFragment>
`;
